helpers/imageUpload.js

```diff
@@ -1,13 +1,14 @@
+const sel = require("../data/selectors.json");
```

Comment:
It looks like the same function already exists in the master

Reply:
it does!